tests/reporting/test_dispute_bond_token_rep_redemption.py

```diff
@@ -17,117 +17,63 @@
 FIRST_REPORTERS_DISPUTE_BOND_AMOUNT = 11 * 10**21
 LAST_REPORTERS_DISPUTE_BOND_AMOUNT = 1375 * 10**20
 
-MARKET_TYPE_CATEGORICAL = 0
 MARKET_TYPE_SCALAR = 1
 
-CATEGORICAL_OUTCOME_A = [10002, 0, 0]
```

Comment:
I got rid of the categorical tests since these take an enormous amount of time and doing both payout types is redundant. If we end up rewriting reporting it will be extra moot too